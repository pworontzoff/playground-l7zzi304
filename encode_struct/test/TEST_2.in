Saperlipopette
Clarisse
rue de la motte
502A
Bruxelles
254
235
1000
24/13/2000
30/2/1999
29/2/1999
12/9/1999
