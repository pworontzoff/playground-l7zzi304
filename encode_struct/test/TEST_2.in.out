Saperlipopette Clarisse
rue de la motte 502A
1000 Bruxelles
Date de naissance: 12/09/1999
